qemu/tests/cfg/rng_read_longtime.cfg

```diff
@@ -0,0 +1,20 @@
+- rng_read_longtime:
+    type = rng_read_longtime
+    kill_vm = yes
+    virt_test_type = qemu
+    test_duration = 36000
```

Comment:
which default value is better, 36000 or 60?
There's a large gap between them.

FYI, params.get("test_duration", "60")